infor-design/enterprise
# 2607 - Fix a performance issue tabbing in IE  [v4.20.x]

**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR improves performance on large modals when tabbing. And also fixed a focus issue in editor.

**Related github/jira issue (required)**:
Fixes #2607 

**Steps necessary to review your pull request (required)**:
Main Test
- Load IE11 (and edge)
- go to http://localhost:4000/components/modal/test-performance.html
- open the modal
- make sure as you tab and shift + tab to the next fields in the form its fairly fast (used to take 2-3 seconds)
- note that the form will still load slow in IE nothing i can do about that


Test One
- any browser
- go to http://localhost:4000/components/modal/example-index.html
- open the modal
- make sure tab and shift + tab is contained

Test Two
- any browser
- go to http://localhost:4000/components/modal/test-dynamic-content.html
- open the modal
- make sure tab and shift + tab is contained
- select the dropdown to add more form content
- make sure tab and shift + tab is still contained

Test Three
- any browser
- go to http://localhost:4000/components/editor/test-modal.html
- open the modal
- make sure tab and shift + tab is contained

**Included in this Pull Request**:
